semopenalex-app/data/templates/Template%3Ahttps%3A%2F%2Fsemopenalex.org%2Fclass%2FWork.html

```diff
@@ -1,76 +1,357 @@
-       {{#each bindings}}
-        <h1> {{title.value}} </h1>
-        
-          
-         
-          <div style='display: flex;    '>
-                             
-            <div style=' width: 70%;'>
-            <h3>by
-              <semantic-query
-                query="Select * Where {
-        Bind(<[[this]]> as ?Work)
-        ?Work <https://semopenalex.org/property/hasAuthorPosition>/<https://semopenalex.org/property/hasAuthor> ?author.
-
-}"
-                template='{{>authors}}'
-              >
-
-                <template id='authors'>{{#each bindings}}<semantic-link iri='{{author.value}}'></semantic-link> {{/each}}</template>
-              </semantic-query>
-            </h3>
-            
-           <h3>DOI {{doi.value}}</h3>
-            <h3>Abstract </h3>
-            {{abstract.value}}
+    query="SELECT * WHERE {
+      BIND(<{{page-resource}}> as ?work) .
+      ?work <https://semopenalex.org/property/citedByCount> ?citedByCount .
+      OPTIONAL { 
+        ?work <http://prismstandard.org/namespaces/basic/2.0/publicationDate> ?publicationDate .
+      }
+      OPTIONAL {
+        ?work <http://purl.org/spar/fabio/hasPublicationYear> ?publicationYear .  
+      }
+      OPTIONAL { 
+        ?work <http://prismstandard.org/namespaces/basic/2.0/doi> ?doi .
+      }              
+      OPTIONAL {
+        ?work <https://semopenalex.org/property/hasHostVenue> ?hostVenue .
+        ?hostVenue <https://semopenalex.org/property/hasVenue> ?venue .
+        ?venue <http://purl.org/dc/terms/publisher> ?publisher .
+      }
+      OPTIONAL {
+        ?work <https://semopenalex.org/property/hasVolume> ?volume .
+      }
+      OPTIONAL {
+        ?work <https://semopenalex.org/property/hasIssue> ?issue .
+      }
+    }" template='{{>tmp}}'>
+    <template id='tmp'>
+      {{#each bindings}}
+        <div class="row cards">
+          <div class="card citations-chart">
+            <div class="publication_year_citations">
+              <div class="publication_year">
+                <div class="publication_year__label">Published in</div>
+                <div class="publication_year__value">
+                  {{publicationYear.value}}
+                </div>
+              </div>
+              <div class="citations">
+                <div class="citations__label">Citations</div>
+                <div class="citations__value">{{citedByCount.value}}</div>                  
+              </div>
+            </div>
+            <semantic-chart type="bar" query="
+              SELECT ?year ?citations WHERE {
+                <{{page-resource}}> <https://semopenalex.org/property/countsByYear> ?count .
+                ?count <https://semopenalex.org/property/year> ?year .
+                ?count <https://semopenalex.org/property/citedByCount> ?citations .
+              } ORDER BY ASC(?year)
+            " sets='[{"dataSetName": "Citations", "category": "citations", "x": "year", "y": "citations"}]'
```

Comment:
If you want to define `x` and `y` then `"category"` should be `year` not `citations`